### PR TITLE
Added partial class ProductController.Comment

### PR DESCRIPTION
**Pull request by Botirjon29**
review qivoradigan joyi

### CODE REVIEW

File: FurnitureShop/src/FurnitureShop.Client.Api/Services/ProductCommentService.cs [hunk 1]
```diff
@@ -62,15 +62,16 @@ public async Task<ProductCommentView> GetProductCommentByIdAsync(Guid commentId)
...
     }
 
-    public async Task AddProductCommentAsync(CreateProductCommentDto commentDto)
+    public async Task<ProductCommentView> AddProductCommentAsync(CreateProductCommentDto commentDto)
     {
         var existingProductComment = new ProductComment
         {
             Comment = commentDto.Comment,
             ParentId = commentDto.ParentId
         };
```
Comment (javlonfattoev) [inline FurnitureShop/src/FurnitureShop.Client.Api/Services/ProductCommentService.cs:71]: userid, createdat ...

File: FurnitureShop/src/FurnitureShop.Client.Api/Controllers/ProductsController.Comment.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    [ProducesResponseType(typeof(List<ProductCommentView>), StatusCodes.Status200OK)]
+    public async Task<ActionResult<List<ProductCommentView>>> GetProductCommentsAsync()
+         => await _productCommentService.GetProductCommentsAsync();
+
+    [HttpGet("{commentId:guid}")]
+    [ProducesResponseType(typeof(ProductCommentView), StatusCodes.Status200OK)]
+    public async Task<ActionResult<ProductCommentView>> GetProductCommentByIdAsync(Guid commentId)
+        => await _productCommentService.GetProductCommentByIdAsync(commentId);
+
+    [HttpPost]
```
Comment (javlonfattoev) [inline FurnitureShop/src/FurnitureShop.Client.Api/Controllers/ProductsController.Comment.cs:20]: post: products/12/comments

File: FurnitureShop/src/FurnitureShop.Client.Api/Controllers/ProductsController.Comment.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+public partial class ProductsController
+{
+
+    [HttpGet("productComments")]
+    [ProducesResponseType(typeof(List<ProductCommentView>), StatusCodes.Status200OK)]
+    public async Task<ActionResult<List<ProductCommentView>>> GetProductCommentsAsync()
+         => await _productCommentService.GetProductCommentsAsync();
+
+    [HttpGet("{commentId:guid}")]
+    [ProducesResponseType(typeof(ProductCommentView), StatusCodes.Status200OK)]
```
Comment (javlonfattoev) [inline FurnitureShop/src/FurnitureShop.Client.Api/Controllers/ProductsController.Comment.cs:16]: get: products/12/comments/15

File: FurnitureShop/src/FurnitureShop.Client.Api/Controllers/ProductsController.Comment.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+using FurnitureShop.Client.Api.ViewModel;
+using Microsoft.AspNetCore.Mvc;
+
+namespace FurnitureShop.Client.Api.Controllers;
+
+public partial class ProductsController
+{
+
+    [HttpGet("productComments")]
+    [ProducesResponseType(typeof(List<ProductCommentView>), StatusCodes.Status200OK)]
```
Comment (javlonfattoev) [inline FurnitureShop/src/FurnitureShop.Client.Api/Controllers/ProductsController.Comment.cs:11]: [HttpGet("{productId:guid}/comments")]

**Review by javlonfattoev — CHANGES_REQUESTED** [review]

**Review by Jamoliddin0710 — APPROVED** [review]

**Pull request closed without merging**